rhessys/util/readtag_worldfile.c

```diff
@@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
 	    if(strcmp(strbuf2,"n_basestations")==0){
 	      break;
 	    }
+	    
```

Comment:
Okay, I tested the code on an ~860 MB worldfile and my hypothesis was wrong.  The new code, as written, is actually quite a bit faster than the previous code (~12 second vs. 30 second startup).  So don't waste any time adding in a more complicated data structure!